filesystem/squashfs_superblock.ksy

```diff
@@ -0,0 +1,112 @@
+meta:
+  id: squashfs_superblock
+  title: SquashFS superblock
+  file-extension:
+    - squashfs
+    - snap
+  endian: le
+  license: CC-0
+  xref:
+    wikidata: Q389314
```

Comment:
```suggestion
```

See https://github.com/kaitai-io/kaitai_struct_formats/pull/308#discussion_r550780445

Reply:
By the above comment, I meant that the `xref` key should be under `file-extension` and above `license` (see the [style guide](https://doc.kaitai.io/ksy_style_guide.html#meta)), and given that I moved it above (https://github.com/kaitai-io/kaitai_struct_formats/pull/308/files#r550780445), this one can be removed.